OHIF/Viewers
# docs(ohif-cli.md): update ohif-cli.md

### Context



### Changes & Results

- Corrected `ohif-cli.md`

### Testing


- open a the docs
- visit a page `development/ohif-cli`
- check the section **create-extension**

### Checklist

#### PR



  semantic-release format and guidelines.

#### Code

  etc.)

#### Public Documentation Updates



  additions or removals.

#### Tested Environment

- [x] Node version: 16.17.1
- [x] Browser: Chrome 114.0.5735.90


[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch
